app/src/main/java/org/fossasia/openevent/general/event/Ticket.kt

```diff
@@ -0,0 +1,31 @@
+package org.fossasia.openevent.general.event
+
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.PrimaryKey
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import com.github.jasminb.jsonapi.IntegerIdHandler
+import com.github.jasminb.jsonapi.annotations.Id
+import com.github.jasminb.jsonapi.annotations.Type
+
+@Type("ticket")
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy::class)
+@Entity
+data class Ticket(
+        @Id(IntegerIdHandler::class)
+        @PrimaryKey
```

Comment:
The event relation is gone

Reply:
@iamareebjamal I dont understand, there wasn't any relation to event before.

Reply:
There should be

Reply:
Okay :+1:

Reply:
Added relationship with event :+1:

Reply:
Foreign Key should be added as well